ops/scripts/ansible_init.sh

```diff
@@ -121,3 +122,8 @@ ssh-keygen -R $webapp_ip
 ssh-keyscan -t ecdsa $bastion_ip >> ~/.ssh/known_hosts
 new_host=$(ssh -i $aws_ssh_key centos@$bastion_ip ssh-keyscan -t ecdsa $webapp_ip)
 echo $new_host  >> ~/.ssh/known_hosts
+
+# Bootstrap playbook
+echo "Saving EC2 IP addresses to GitLab"
+env TF_KEY_NAME=private_ip ansible-playbook -i ../../inventories bootstrap_playbook.yml --tags="webapp_ips" --extra-vars="gigadb_env=$target_environment"
+ansible-playbook -i ../../inventories bootstrap_playbook.yml --tags="bastion_ips" --extra-vars="gigadb_env=$target_environment"
```

Comment:
Hi @rija,

I am think the bootstrap playbook can only be executed 1 time only if the `--tags` option is removed, like this:
```
env TF_KEY_NAME=private_ip ansible-playbook -i ../../inventories bootstrap_playbook.yml --extra-vars="gigadb_env=$target_environment"
```
And the output will be:
```
PLAY RECAP ************************************************************************************************************************************************************************************************************************
webapp_private_ip                : ok=5    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
bastion_private_ip                : ok=5    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0 
```
The only difference is that user needs to allow the connection to bastion server through private IP at the first time.

I understand why you would execute the `ansible-playbook` 2 times, as bastion server does not need private ip for connection, but it would help by reducing the execution time and have a more informative `PLAY RECAP` at the end.

It is just a thought and I would like to see what you think.

Reply:
Hi @kencho51,

I understand what's bothering you with this approach, and it does annoy me too.
It's a limitation of the dynamic inventory script we are using that can only return either private ips for all hosts or public ips for all hosts, not mix-and-match.
I should have probably listed that as a "known issue" too.

1) In the case of this PR, the Ansible playbook calls are not meant to be made manually and separately, and performance overhead is negligible, so I deem the situation acceptable


2) The most promising solution to the issue involves changing the dynamic inventory script, but:
* that would make this PR bigger than it should
* it should be done as a way to also fix the need to set `TF_KEY_NAME` env to the `webapp_playbook.yml` as the root cause is the same
* If solution is still built on the open source terraform-inventory project we are using, we should aim to submit a PR to that project

So, because of 1)  and 2), better leave this for another time